src/message_bus.zig

```diff
@@ -136,25 +136,28 @@ fn MessageBusImpl(comptime process_type: ProcessType) type {
             context: Context,
             comptime on_message: fn (context: Context, message: *Message) void,
         ) void {
+            // TODO This looks like a bug in the compiler, not seeing on_message as used?
+            _ = on_message;
+
```

Comment:
I can't reproduce this error locally, I think we can remove this.
```suggestion
```

Reply:
I can't reproduce it any more now either with a new install of 0.9.0.